### PR TITLE
fix(api): Fix Cloud Dedicated /ping descriptions

### LINKED ISSUE #5610: > the /ping endpoint in dedicated only shows querier status
> the /ping endpoint in the dedicated contour ingress configuration points to the querier. i'll go see if i can correlate these to querier crashes or restarts (i've been looking at the router)

> https://docs.influxdata.com/influxdb/cloud-dedicated/api/v2/#operation/GetPing
> 
> Retrieves the status and InfluxDB version of the instance.
> 
> Use this endpoint to monitor uptime for the InfluxDB instance. The response returns a HTTP 204 status code to inform you the instance is available.

@sanderson I think we should remove this from our docs as this is not an accurate description of what `/ping` in dedicated is currently doing

_Originally posted by @helenosheaa in https://github.com/influxdata/EAR/issues/5473#issuecomment-2338873503_

### PR DESCRIPTION
**Pull request by jstirnaman**
- Cloud Dedicated /ping only reports querier, not ingester, health

Closes #5610

### CODE REVIEW

File: api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml [hunk 1]
```diff
@@ -240,6 +240,83 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /ping:
```
Comment (jstirnaman) [inline api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml:243]: @helenosheaa Only review this one. Any corrections here, I'll paste to the others.

File: api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml [hunk 2]
```diff
@@ -240,6 +240,83 @@ paths:
...
+    get:
+      description: |
+        Reports the InfluxQL bridge querier health and the InfluxDB version of the instance.
+        
+        The response is a HTTP `204` status code to inform you the querier is available.
+
+        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; doesn't check the status of ingesters.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml:250]:
```suggestion
        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; not the status of ingesters.
```

File: api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml [hunk 3]
```diff
@@ -240,6 +240,83 @@ paths:
...
+
+        To check the health of ingesters before writing data, send a request to one of the [write endpoints](/influxdb/cloud-dedicated/api/v2/#tag/Write).
+
+        This endpoint doesn't require authentication.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml:254]: At first, I thought this was referring to the write endpoint referenced above.

```suggestion
        The `/ping` endpoint doesn't require authentication.
```

File: api-docs/cloud-dedicated/v2/ref.yml [hunk 4]
```diff
@@ -149,17 +149,20 @@ paths:
...
     get:
       description: |
-        Retrieves the status and InfluxDB version of the instance.
+        Reports the InfluxQL bridge querier health and the InfluxDB version of the instance.
+        
+        The response is a HTTP `204` status code to inform you the querier is available.
 
-        Use this endpoint to monitor uptime for the InfluxDB instance. The response
-        returns a HTTP `204` status code to inform you the instance is available.
+        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; doesn't check the status of ingesters.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v2/ref.yml:156]:
```suggestion
        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; not the status of ingesters.
```

File: api-docs/cloud-dedicated/v2/ref.yml [hunk 5]
```diff
@@ -149,17 +149,20 @@ paths:
...
+
+        To check the health of ingesters before writing data, send a request to one of the [write endpoints](/influxdb/cloud-dedicated/api/v2/#tag/Write).
 
         This endpoint doesn't require authentication.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v2/ref.yml:160]:
```suggestion
        The `/ping` endpoint doesn't require authentication.
```

File: api-docs/cloud-dedicated/v2/ref.yml [hunk 6]
```diff
@@ -184,17 +187,20 @@ paths:
         - Ping
     head:
       description: |
-        Returns the status and InfluxDB version of the instance.
+        Reports the InfluxQL bridge querier health and the InfluxDB version of the instance.
+        
+        The response is a HTTP `204` status code to inform you the querier is available.
+
+        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; doesn't check the status of ingesters.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v2/ref.yml:194]:
```suggestion
        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; not the status of ingesters.
```

File: api-docs/cloud-dedicated/v2/ref.yml [hunk 7]
```diff
@@ -184,17 +187,20 @@ paths:
...
 
-        Use this endpoint to monitor uptime for the InfluxDB instance. The response
-        returns a HTTP `204` status code to inform you the instance is available.
+        To check the health of ingesters before writing data, send a request to one of the [write endpoints](/influxdb/cloud-dedicated/api/v2/#tag/Write).
 
         This endpoint doesn't require authentication.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v2/ref.yml:198]:
```suggestion
        The `/ping` endpoint doesn't require authentication.
```

File: api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml [hunk 8]
```diff
@@ -240,6 +240,83 @@ paths:
...
+      summary: Get the status of the instance
+      tags:
+        - Ping
+    head:
+      description: |
+        Reports the InfluxQL bridge querier health and the InfluxDB version of the instance.
+        
+        The response is a HTTP `204` status code to inform you the querier is available.
+
+        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; doesn't check the status of ingesters.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml:288]:
```suggestion
        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; not the status of ingesters.
```

File: api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml [hunk 9]
```diff
@@ -240,6 +240,83 @@ paths:
...
+
+        To check the health of ingesters before writing data, send a request to one of the [write endpoints](/influxdb/cloud-dedicated/api/v2/#tag/Write).
+
+        This endpoint doesn't require authentication.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml:292]:
```suggestion
        The `/ping` endpoint doesn't require authentication.
```

**Review by sanderson — APPROVED** [review]
Just some minor suggestions.

**Pull request merged**